Tools/nib2cib/NSBox.j

```diff
@@ -80,6 +66,33 @@
     return self;
 }
 
+// Override
+
+- (CGRect)_nib2CibAdjustment
+{
+    if ((_boxType !== CPBoxSeparator) && ((_boxType === CPBoxPrimary) || (_boxType === CPBoxSecondary)))
```

Comment:
@didierkorthoudt One last thing..... I hope 😄.... Why does this `if` first test `_boxType !== CPBoxSeparator` and after that `(_boxType === CPBoxPrimary) || (_boxType === CPBoxSecondary)`. Is the first test really needed or am I missing something here?

Reply:
You're absolutely right, I've just copied the old test and should have seen this… I'll adapt, thank you !